src/Settings/Mods.ts

```diff
@@ -316,6 +316,28 @@ export function GenerateModsSettings(): GameStartr.IModAttachrCustoms {
                         }
                     }
                 }
+            },
+            {
+                name: "Mobile PokeCenter",
+                enabled: false,
+                events: {
+                    onModEnable: function (mod: ModAttachr.IMod, eventName: string, args: any[]): void {
+                        const party: IPokemon[] = this.ItemsHolder.getItem("PokemonInParty");
+                        const onChange: (enabled: boolean) => void = args[1][0];
+
+                        for (const pokemon of party) {
+                            this.battles.healPokemon(pokemon);
+                        }
+
+                        this.menus.displayMessage("Your party has been fully healed!");
+                        this.ModAttacher.toggleMod("Mobile PokeCenter");
+                        // Call onChange with true because UserWrappr flips it right after.
+                        onChange(true);
```

Comment:
...and you shouldn't need to explain what true is in an inline comment. That should be covered by the parameter description.